packages/devtools/README.md

```diff
@@ -1,78 +1,66 @@
 [![Build Status](https://travis-ci.org/flutter/devtools.svg?branch=master)](https://travis-ci.org/flutter/devtools)
 
-## What is this?
+## Trying it out
 
-This repo is a companion repo to the main [flutter repo](https://github.com/flutter/flutter).
-It contains the source code for a suite of performance tools for Dart and Flutter.
+### Installing
 
-## But there's not much here?
+Currently, the best way to try DevTools out is by running it as a pub globally activated package.
 
-It's still very early in development - stay tuned.
+If you have `pub` on your path, you can run:
 
-## Development
+- `pub global activate devtools`
 
-- `git clone https://github.com/flutter/devtools`
-- `cd devtools/packages/devtools`
-- `pub get`
+If you have `flutter` on your path, you can run:
 
-From a separate terminal:
-- `cd <path/to/flutter-sdk>/examples/flutter_gallery`
-- ensure the iOS Simulator is open (or a physical device is connected)
-- `flutter run`
+- `flutter packages pub global activate devtools`
 
-From the packages/devtools directory:
-- `pub run webdev serve web`
+That will  install (or update) DevTools on your machine.
 
-Then, open a browser window to the local url specified by webdev. After the page has loaded, append
-`?port=xxx` to the url, where xxx is the port number of the service protocol port, as specified by 
-the `flutter run` output.
+Going forward, we expect to have additional (and easier!) distributions mechanisms for DevTools.
 
-For more productive development, launch your flutter application specifying
-`--observatory-port` so the observatory is available on a fixed port. This
-lets you avoid manually entering the observatory port parameter each time
-you launch the application.
+### Run the DevTools application server
 
-- `flutter run --observatory-port=8888`
-- `open http://localhost:8080/?port=8888`
+Next, you want to run the local web server which can serve the DevTools application itself.
```

Comment:
nits:
- remove "you want to"
- comma after "server"

Reply:
done